core/templates/components/rubrics-editor/rubrics-editor.component.ts

```diff
@@ -95,6 +95,39 @@ export class RubricsEditorComponent {
     return Boolean(this.editableExplanations[difficulty][index]);
   }
 
+  ismediumLevelExplanationLengthEqualsOne(difficulty: string): boolean {
+    if (difficulty === this.skillDifficultyMedium) {
+      return this.rubrics[1]._explanations.length === 1;
+    }
+    return false;
+  }
+
+  isMediumLevelExplanationValid(): boolean {
+    if (this.rubrics[1]._explanations.length > 1) {
+      return true;
+    }
+    return this.editableExplanations[this.skillDifficultyMedium][0].length > 0;
```

Comment:
If there are 0 medium level explanations, won't this `return` statement lead to an error? Since `this.editableExplanations[this.skillDifficultyMedium] = []`? I may be wrong here.

Reply:
Yes you are right, this will through us an error. What we can do is we can add an `else if` in which we can check if the length is one else we can return false.
Thanks

Reply:
Or we can completely remove the length part we are checking because we are letting the user delete the explanation and simply check the total no of explanations.
Thanks

Reply:
Done